Admin/pages/AdminApproval.php

```diff
@@ -35,13 +35,15 @@ protected function initInternal()
 
 		$this->pending_ids = $this->getPendingIds();
 
-		if (count($this->pending_ids) == 0)
+		if (count($this->pending_ids) === 0) {
 			$this->relocate();
+		}
 
 		$this->id = $this->app->initVar('id');
 
-		if ($this->id === null)
+		if ($this->id === null) {
 			$this->id = $this->getNextId();
+		}
```

Comment:
I was curious about this part. If `this->id === null` then won't `$this->getNextId()` always return `null` too?